modin/core/storage_formats/pandas/groupby.py

```diff
@@ -87,12 +93,16 @@ def build_qc_method(cls, agg_name):
         """
         map_fn, reduce_fn, d2p_fn = cls.get_impl(agg_name)
         map_reduce_method = GroupByReduce.register(
-            map_fn, reduce_fn, default_to_pandas_func=d2p_fn
+            map_fn, reduce_fn, default_to_pandas_func=d2p_fn, finalizer_fn=finalizer_fn
         )
 
         def method(query_compiler, *args, **kwargs):
-            if ExperimentalGroupbyImpl.get():
+            if allow_experimental_groupby and ExperimentalGroupbyImpl.get():
```

Comment:
Why not? (without new parameter)
```suggestion
            if finalizer_fn is None and ExperimentalGroupbyImpl.get():
```

Reply:
sure, removed the parameter